public/video-ui/src/components/Videos/ComposerPageCreate.js

```diff
@@ -12,10 +12,14 @@ export default class ComposerPageCreate extends React.Component {
     return getStore().getState().config.composerUrl;
   };
 
+  getComposerId = () => getStore().getState().usage.data.preview.video[0].fields.internalComposerCode;
```

Comment:
this'll fail if there's a published usage, could we check if `preview.video.length === 0` and then check `published.video.length`?